Makefile.ci

```diff
@@ -24,9 +24,7 @@ CI_TARGETS=ci-all \
 
 # Generic rule, we use make to easy travis integraton with mixed rules
 $(CI_TARGETS): ci-%:
-	rm -f ci-$*.ok
-	+(./dev/ci/ci-$*.sh 2>&1 && touch ci-$*.ok) | tee time-of-build.log
+	+./dev/ci/ci-pipe-tee.sh ./dev/ci/ci-$*.sh time-of-build.log
 	echo 'Aggregating timing log...' && echo -en 'travis_fold:start:coq.test.timing\\r'
 	python ./tools/make-one-time-file.py time-of-build.log
 	echo -en 'travis_fold:end:coq.test.timing\\r'
```

Comment:
If I understand correctly before when there was a failure the above python script was still called but after this it isn't. Is this desired?
Also it might be more beautiful to have some `ci-wrapper.sh` script which does the entire recipe rather than `ci-pipe-tee.sh` doing part of it, but that's just my sense of aesthetics.